src/wallet/wallet.cpp

```diff
@@ -2311,18 +2311,24 @@ DBErrors CWallet::LoadWallet()
     return nLoadWalletRet;
 }
 
-DBErrors CWallet::ZapSelectTx(std::vector<uint256>& vHashIn)
+void CWallet::ClearTxns(const std::vector<uint256>& tx_hashes)
 {
-    AssertLockHeld(cs_wallet);
-    DBErrors nZapSelectTxRet = WalletBatch(GetDatabase()).ZapSelectTx(vHashIn);
-    for (const uint256& hash : vHashIn) {
+    for (const uint256& hash : tx_hashes) {
         const auto& it = mapWallet.find(hash);
         wtxOrdered.erase(it->second.m_it_wtxOrdered);
-        for (const auto& txin : it->second.tx->vin)
+        for (const auto& txin: it->second.tx->vin)
```

Comment:
In 559529480fedd4181dab76f4452b5998d0cdadc3 "refactor: decouple wallet EraseTxns from ZapSelectTx"

Unnecessary whitespace change.

Reply:
fixed